docs/src/pages/getting-started/usage/usage.md

```diff
@@ -39,7 +39,7 @@ Material-UI usage experience can be improved with a handful of important globals
 
 ### Responsive meta tag
 
-Material-UI is developed mobile first, a strategy in which we first write code for mobile devices and then scale up components as necessary using CSS media queries.
+Material-UI is developed mobile-first, a strategy in which we first write code for mobile devices and then scale up components as necessary using CSS media queries.
```

Comment:
```suggestion
Material-UI is developed mobile-first, a strategy in which we first write code for mobile devices, and then scale up components as necessary using CSS media queries.
```

I'm not sure that's entirely true, but lets focus on fixing the grammar, and fix the facts separately. 😄

Reply:
I can't say I do this 😄